src/aiida/cmdline/commands/cmd_process.py

```diff
@@ -618,8 +627,17 @@ def process_dump(
         include_extras=include_extras,
         overwrite=overwrite,
         flat=flat,
+        also_remote=also_remote,
     )
 
+    if also_remote:
+        echo.echo_report(
+            '`--also-remote` set to True. Will try to retrieve additional files from the `workdir` of the remote Computer.'  # noqa: E501
+        )
+        echo.echo_report(
+            'If files are non-existent, they will be skipped silently. Check if the output files are what you expect.'
```

Comment:
Is this a good idea? will be difficult to tell if they werent copied because they no longer exist or because there was a problem